app/javascript/src/NewService/components/FormTemplate.jsx

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import React from 'react'
+import {CSRFToken} from 'utilities/utils'
+import type {FormTemplateProps} from 'NewService/types'
+
+type ServiceDiscoveryFormTemplateProps = {
+  hiddenServiceDiscoveryInput: {
+    value: string,
+    name: string,
+    id: string
+  },
+  serviceDiscoverySelect: {
+    [string]: {
+        htmlFor: string,
+        label: string,
+        name: string,
+        id: string
+    }
+  }
+}
+
+const serviceDiscoveryProps: ServiceDiscoveryFormTemplateProps = {
+  hiddenServiceDiscoveryInput: {
+    value: 'discover',
+    name: 'service[source]',
+    id: 'service_source'
+  },
+  serviceDiscoverySelect: {
+    namespace: {
+      htmlFor: 'namespace',
+      label: 'Namespace',
+      name: 'service[namespace]',
+      id: 'service_namespace'
+    },
+    projects: {
```

Comment:
Why "projects"? This was supposed to be named "service", no? (It's "namespace" that I previously suggested to be renamed to "project" BTW.)